cellpack/autopack/DBRecipeHandler.py

```diff
@@ -561,6 +563,76 @@ def upload_job_status(self, job_id, status, result_path=None, error_message=None
                 },
             )
 
+    def upload_outputs_to_s3(self, output_folder, recipe_name, job_id):
+        """
+        Upload packing outputs to S3 bucket
+        """
+
+        bucket_name = "cellpack-results"
+        region_name = "us-west-2"
+        sub_folder_name = "runs"
+        s3_prefix = f"{sub_folder_name}/{recipe_name}/{job_id}"
+
+        try:
+            handler = DATABASE_IDS.handlers().get(DATABASE_IDS.AWS)
+            aws_handler = handler(
+                bucket_name=bucket_name,
+                sub_folder_name=sub_folder_name,
+                region_name=region_name,
+            )
+
+            upload_result = aws_handler.upload_directory(
+                local_directory_path=output_folder, s3_prefix=s3_prefix
+            )
+
+            if upload_result["success"]:
+                # generate public URLs for the uploaded files
+                base_url = f"https://{bucket_name}.s3.{region_name}.amazonaws.com"
+                public_urls = [
+                    f"{base_url}/{file_info['s3_key']}"
+                    for file_info in upload_result["uploaded_files"]
+                ]
+                outputs_directory = f"https://us-west-2.console.aws.amazon.com/s3/buckets/{bucket_name}/{s3_prefix}/"
+                DBUploader._update_outputs_directory(job_id, outputs_directory)
+
+                logging.info(
+                    f"Successfully uploaded {upload_result['total_files']} files to {outputs_directory}"
+                )
+                logging.debug(f"Total size: {upload_result['total_size']:,} bytes")
+                logging.debug(f"Public URL base: {base_url}/{s3_prefix}/")
+
+                return {
+                    "success": True,
+                    "run_id": job_id,
+                    "s3_bucket": bucket_name,
+                    "s3_prefix": s3_prefix,
+                    "public_url_base": f"{base_url}/{s3_prefix}/",
+                    "uploaded_files": upload_result["uploaded_files"],
+                    "total_files": upload_result["total_files"],
+                    "total_size": upload_result["total_size"],
+                    "urls": public_urls,
+                }
+        except Exception as e:
+            error_msg = f"Failed to upload packing results to S3: {e}"
+            logging.error(error_msg)
+            return {"success": False, "error": error_msg}
+
+    def _update_outputs_directory(job_id, outputs_directory):
+        # Update firebase with the new outputs directory
+        handler = DATABASE_IDS.handlers().get("firebase")
+        initialized_db = handler(default_db="staging")
+        if job_id:
+            initialized_db.update_or_create(
```

Comment:
nit: should we update the timestamp field in the job_Status collection here too?